pkg/features/isis/collector.go

```diff
@@ -3,26 +3,32 @@
 package isis
 
 import (
+	"github.com/pkg/errors"
```

Comment:
Imports should be sorted:
1. standard library
2. project internal packages
3. project external packages
4. renamed packages (first internal, then external)